packages/client/test/integration/test-node/test/compatibility/compatibility.spec.ts

```diff
@@ -0,0 +1,242 @@
+/**
+ * This file is for Compatibility Matrix tests, which assigns specific
+ * Allure context tags.
+ */
+
+import * as allure from 'allure-vitest'
+import { pipe } from 'fp-ts/function'
+import { describe, expect, TaskContext, test } from 'vitest'
+import { clientFactory, setupPeerTestsLifecycle } from '../util'
+import { datamodel, sugar } from '@iroha2/data-model'
+import { CLIENT_CONFIG } from '@iroha2/test-configuration'
+import { Seq } from 'immutable'
+
+setupPeerTestsLifecycle()
+
+async function annotateAllure(
```

Comment:
You can also move it to shared utils and re-use it in all other files.

Reply:
ready